lambdas/ec2/hipchat_notifier.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+import boto3
+import json
+import ast
+import zlib
+import base64
+import os
+from urllib2 import Request, urlopen
+
+HIPCHAT_TOKEN = 'LOCAL_TOKEN'
+HIPCHAT_ROOM_ID = 'LOCAL_ID'
+
+def get_account_alias():
+    """
+    Return the first alias listed from Amazon.
+    """
+    client = boto3.client('iam')
+    return client.list_account_aliases()['AccountAliases'][0]
+
+def read_token():
+    """
+    Read in the environment HIPCHAT_TOKEN, or read the local token.
+    """
+
+    if 'HIPCHAT_TOKEN' in os.environ:
+        return os.environ['HIPCHAT_TOKEN']
+    else:
+        return HIPCHAT_TOKEN    
```

Comment:
@tvpartytonight the fact that we're returning a default value here means that the failure behavior of the default not being an actual hipchat value (which I'm not suggesting we make into a working value) is that our post later on in execution should fail, returning something other than the 204 code that we're looking for.

Since we know at this point what the context for the failure will be, why don't we fail here, rather than waiting for all execution between here and the post plus the network time of the post to happen, requiring us to either a) hope that the return json tells us that the tokens are bad, or b) need to track the error from AWS back to the token ourselves? (ref: [Handle Errors in Context](http://wiki.c2.com/?HandleErrorsInContext))